### ISSUE TITLE
Headless chrome not recognized correclty

### ISSUE THREAD

**Issue by LarsDenBakker** [Polymer/browser-capabilities#3]
Headless Chrome has a different user agent than regular chrome. This causes it to not be recognized.

Adding a headless chrome entry just like Chrome/Chromium would solve this. https://github.com/Polymer/browser-capabilities/blob/master/src/browser-capabilities.ts#L37

However it might need to be solved in a more generic way.

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/64.0.3240.0 Safari/537.36

**Comment by justinfagnani** [comment]
ua-parser-js assed Headles Chrome support in: https://github.com/faisalman/ua-parser-js/commit/b8978f5933d132422923881a7b8dfde17bb5f3e7

Looks like it might be coming back as a different browser name though: "Chrome Headless"

**Comment by paales** [comment]
us-parser-js correctly recognizes the user agent but here:

https://github.com/Polymer/browser-capabilities/blob/master/src/browser-capabilities.ts#L37
It should also include a 'Chrome Headless' predicate.

```
    'Chrome Headless': {
        es2015: since(60),
        push: since(60),
        serviceworker: since(60),
        modules: since(60),
    },
```

**Comment by aomarks** [comment]
Fixed and now released in `0.2.2`.
